test/integration/base_integration_test.cc

```diff
@@ -54,6 +54,9 @@ BaseIntegrationTest::BaseIntegrationTest(const InstanceConstSharedPtrFn& upstrea
       version_(version), upstream_address_fn_(upstream_address_fn),
       config_helper_(version, *api_, config),
       default_log_level_(TestEnvironment::getOptions().logLevel()) {
+#ifdef ENVOY_USE_NEW_HTTP1_PARSER_IN_INTEGRATION_TESTS
+  config_helper_.addRuntimeOverride("envoy.reloadable_features.enable_new_http1_parser", "true");
+#endif
```

Comment:
I take it that all existing tests pass with the new parser?  Are there any known differences between the two parsers?

Reply:
I added some description, there shouldn't be any difference from user perspective except the one relating to the lenient header comment.